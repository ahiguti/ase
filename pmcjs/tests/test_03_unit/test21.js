
/* tied scalar */

Perl.Use("TieTest");
var z = Perl.NewTiedScalar("TieTest::STie", 1, 0, 0);
print("z=" + z.Get());
z.Set(5);
print("z=" + z.Get());

