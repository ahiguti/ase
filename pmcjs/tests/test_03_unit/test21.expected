tiescalar fetch
tiescalar store [5]
tiescalar fetch
z=123
z=10
